docs/NMOS-With-IPMX-PEP.md

```diff
@@ -211,27 +243,39 @@ The `urn:x-nmos:cap:transport:privacy` capability MUST NOT allow both `true` and
 
 A Receiver implementing privacy encryption and the PEP protocol MUST provide IS-05 `ext_privacy_*` extended transport parameters and associated constraints that specify the extent of support for the features defined in [TR-10-13][].
 
-A Receiver SHOULD provide a `urn:x-nmos:cap:transport:privacy` capability to indicate support for Senders that use privacy encryption and the PEP protocol. A capability value of `true` indicates that privacy encryption and the PEP protocol are supported, while a value of `false` indicates that they are not supported. A Receiver MAY support either the `true` or `false` value.
+A Receiver MUST provide a `urn:x-nmos:cap:transport:privacy` capability to indicate support for Senders that use privacy encryption and the PEP protocol. A capability value of `true` indicates that privacy encryption and the PEP protocol are supported, while a value of `false` indicates that they are not supported. A Receiver MAY support either the `true` or `false` value.
 
 > Note: A Receiver is not allowed by [TR-10-13][] to support both values. An NMOS API is not allowed to change the state (enabled or disabled) of privacy encryption.
 
 ## Controller
 
-A Controller MUST verify Receivers' compliance with an active Sender using privacy encryption and the PEP protocol. This is achieved by checking the Sender’s SDP transport file for a `privacy` attribute, or by checking the Sender's `privacy` attribute, and by verifying the Sender's IS-05  `ext_privacy_*` extended transport parameters at the `active` endpoint. 
+A Controller MUST verify Receivers' compliance with an active Sender using privacy encryption and the PEP protocol. 
+
+A Controller establishes that an active Sender is using privacy encryption and the PEP protocol by checking the Sender's `privacy` attribute, or by checking the Sender’s SDP transport file for a `privacy` attribute, or by checking the Sender's `urn:x-nmos:cap:transport:privacy` capability, or by verifying the Sender's IS-05  `ext_privacy_*` extended transport parameters at the `active` endpoint.
+
+The Sender's `privacy` attribute being set to `true` or the presence of the `privacy` attribute in the SDP transport file, indicates that the stream is privacy-protected. 
+
+Similarly, the Sender's IS-04 `urn:x-nmos:cap:transport:privacy` capability, enumerating the value `true`, indicates that the stream is privacy-protected. 
+
+Finally, the presence of the Sender's IS-05 `ext_privacy_protocol` and `ext_privacy_mode` transport parameters at the `active` endpoint with a value other than "NULL" indicate that the stream is privacy-protected.
+
+When considering an inactive Sender, a Controller MUST NOT rely on the content of the SDP transport file as it MAY NOT be available until the Sender becomes active.
```

Comment:
Could it be that some devices still have an SDP file (from previous activations) but they don't update it until the sender becomes active?
```suggestion
When considering an inactive Sender, a Controller MUST NOT rely on the content of the SDP transport file as it MAY NOT be available or up to date until the Sender becomes active.
```

Reply:
I agree it is a safer to include "up to date"